package.json

```diff
@@ -30,8 +30,8 @@
   "main": "./lib/npm-check-updates",
   "scripts": {
     "nyc": "nyc",
-    "lint": "eslint .",
-    "lintfix": "eslint --fix .",
+    "lint": "eslint --cache --cache-location node_modules/.cache/.eslintcache --report-unused-disable-directives .",
```

Comment:
Can you add a trailing `/` as recommended in the eslint docs?

> Important note: If the directory for the cache does not exist make sure you add a trailing / on *nix systems or \ in windows. Otherwise the path will be assumed to be a file.

https://eslint.org/docs/user-guide/command-line-interface#cache-location

Reply:
But `.eslintcache` is a file here?

Reply:
Hmmm. Maybe I got confused by the documentation. I'll trust when you have then.

Reply:
I literally meant it to be `node_modules/.cache/.eslintcache` but if you want another location/file, let me know :)